docs/google/floquet.ipynb

```diff
@@ -0,0 +1,1352 @@
+{
```

Comment:
<p>Is it possible to somehow reference https://github.com/quantumlib/Cirq/blob/master/cirq/ops/fsim_gate.py#L193 which shows the unitary model of PhasedFSimGate here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/3765/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>TeX'ed up the matrix.&nbsp;</p>

Reply:
<p>Is it supposed to be that high or is it a bug of this reviewnb tool?</p>

Reply:
<p>Looks like a reviewnb thing, there's only one new line </p>

Reply:
<p>Where it's talking about the simplest correction is for single-qubit Z rotations, can we reference which angle(s) that corrects? </p>

Reply:
@mrwojtek this corrects zeta and gamma, right?

Reply:
<p>This notebook corrects for zeta and gamma.</p><p> </p><p>The make_zeta_chi_gamma_compensation supports also chi but this example doesn't characterize chi, so it is not corrected neither.</p>

Reply:
Great, thanks. Updated and clarified the language in this section, sticking more closely to new function names and discerning between characterizing / compensating.